### PR TITLE
Cleanup scalability test configs

### PR DESCRIPTION
**Pull request by wojtek-t**

### CODE REVIEW

**Comment by krzyzacy** [comment]
I'll let @shyamjvs to stamp

File: jobs/env/ci-kubernetes-e2e-scalability.env [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 REGISTER_MASTER=true
+LOGROTATE_MAX_SIZE=5G
+
+# Use IP-aliases for scalability tests.
 KUBE_GCE_ENABLE_IP_ALIASES=true
 
 # Switch off image puller to workaround #44701
 PREPULL_E2E_IMAGES=false
-# Reduce logs verbosity
-TEST_CLUSTER_LOG_LEVEL=--v=2
```
Comment (shyamjvs) [inline jobs/env/ci-kubernetes-e2e-scalability.env]: Why are we not setting `--v=2` in the common env file?

**Comment by shyamjvs** [comment]
One small comment - LGTM overall.
Also, since you're anyway touching the .env files could you also please organize the variables into categories (i.e cluster-env, kubernetes-env, e2e-env) like [here](https://github.com/kubernetes/test-infra/blob/master/jobs/env/ci-kubernetes-e2e-gce-scale-performance.env)?

File: jobs/env/ci-kubernetes-e2e-scalability.env [hunk 2]
```diff
@@ -1,19 +1,19 @@
 ### job-env
```
Comment (shyamjvs) [inline jobs/env/ci-kubernetes-e2e-scalability.env]: Maybe rename this file to `ci-kubernetes-e2e-gce-scalability-common.env` for more clarity?

File: jobs/env/ci-kubernetes-e2e-scalability.env (hunk 1 above)
Comment (wojtek-t) [reply jobs/env/ci-kubernetes-e2e-scalability.env]: On purpose - different jobs set different verbosity.
Once our configs are clearer, we may think more how to fix that, but for now I don't want to mix that.

File: jobs/env/ci-kubernetes-e2e-scalability.env (hunk 2 above)
Comment (wojtek-t) [reply jobs/env/ci-kubernetes-e2e-scalability.env]: Done

**Comment by wojtek-t** [comment]
PTAL

File: jobs/env/ci-kubernetes-e2e-scalability.env (hunk 1 above)
Comment (shyamjvs) [reply jobs/env/ci-kubernetes-e2e-scalability.env]: ok, sgtm

**Comment by wojtek-t** [comment]
> Also, since you're anyway touching the .env files could you also please organize the variables into categories (i.e cluster-env, kubernetes-env, e2e-env) like here?

As discussed offline - I'm already cleaning this files more in subsequent PRs. So I will do it there.

**Comment by shyamjvs** [comment]
/lgtm

Thanks.

**Comment by shyamjvs** [comment]
Seems like this change broke our 1.10 job - https://k8s-testgrid.appspot.com/sig-scalability-gce#gce-1.10
Because of enabling IP-aliases in the common config (which wouldn't work with custom subnets in <= 1.10 where my fixes don't exist).

**Pull request merged**